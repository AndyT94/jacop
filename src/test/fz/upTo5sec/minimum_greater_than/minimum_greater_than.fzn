var 5..5: var1:: output_var;
var 3..3: var2:: output_var;
array [1..4] of var int: variables:: output_array([1..4]) = [8,5,3,8];
solve  satisfy;
