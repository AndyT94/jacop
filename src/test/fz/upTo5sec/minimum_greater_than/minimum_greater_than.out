var1 = 5;
var2 = 3;
variables = array1d(1..4, [8, 5, 3, 8]);
----------